docs/source/testing/larva/ignores.rst

```diff
@@ -0,0 +1,345 @@
+Ignores
+========
+
+As explained in section :ref:`gettingStartedLarva`, Larva-tests verify if values that are read from the tested Frank configuration are equal to the expected constant values. However, some tested systems produce different results each time that they are executed, even though the inputs are the same. Examples are systems that return a time stamp. In Larva, you can address this issue using ignores. With ignores, values returned from the tested system are transformed, before they are compared to the expected value. As a Frank developer, you have to choose these transformations wisely and retain meaningful aspects of your returned value. 
+
+This section gives an overview of the transformations (ignores) that are available to you. Below is an overview table followed by explanation and examples for every transformation.
+
+.. list-table:: Overview
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Options
+     - Description
+   * - ignoreRegularExpressionKey
+     - Replaces all occurrences of regular expression with
+      
+       IGNORE
+   * - ignoreKey 
+     - Replaces all matching occurrences of provided key with
+      
+       IGNORE  
+   * - ignoreContentBetweenKeys 
+     - Replaces all characters found between key1 and key2,
+      
+       excluding key1 and key2, with IGNORE
+   * - ignoreKeysAndContentBetweenKeys
+     - Replaces all characters found between key1 and key2, 
+       
+       including key1 and key2, with IGNORE
+   * - ignoreContentBeforeKey
+     - Ignores everything before provided key has been found
+   * - ignoreContentAfterKey
+     - Ignores everything after provided key has been found
+   * - ignoreCurrentTimeBetweenKeys       
+     - Ignore current time between key1 and key2
+   * - replaceRegularExpressionKeys
+     - Replaces all matches of regular expression (key1) with
+      
+       the value of key2
+   * - replaceKey
+     - Replaces all matches occurrences of provided key with
+      
+       provided value
+   * - replaceEverywhereKey     
+     - Replaces all matches occurrences of provided key with
+      
+       provided value, same as replaceKey
+   * - removeRegularExpressionKey
+     - Replaces all matches of regular expression with “”
+   * - removeKeysAndContentBetweenKeys
+     - Removes all characters found between key1 and key2,
+      
+       including key1 and key2
+   * - removeKey 
+     - Removes all matching occurrences of provided key
+   * - decodeUnzipContentBetweenKeys
+     - Decode and unzip content between key1 and key2
+   * - canonicaliseFilePathContentBetweenKeys
+     - Canonicalise filepath content between key1 and key2
+   * - formatDecimalContentBetweenKeys
+     - Format decimal content between key1 and key2
+
+
+The ignores use the following structure: ``<ignoreOption><identifier>.key=<text>``.
+
```

Comment:
I disagree with the word "identifier" here. An identifier is a string that is like the name of a variable. But here you mean either a number or a name preceded by a dot. I would not say &lt;identifier&gt; but &lt;numberOrName&gt;. If you do it like this, the reader is prepared when she reads about the two ways to write ignores.

Reply:
I read the text again. Now it is OK.